diff --git a/src/.vs/src/v16/.suo b/src/.vs/src/v16/.suo
index 5e5ed43..e2d56d8 100644
Binary files a/src/.vs/src/v16/.suo and b/src/.vs/src/v16/.suo differ
diff --git a/src/.vs/src/v16/Server/sqlite3/storage.ide b/src/.vs/src/v16/Server/sqlite3/storage.ide
index c4abec3..0f83e8d 100644
Binary files a/src/.vs/src/v16/Server/sqlite3/storage.ide and b/src/.vs/src/v16/Server/sqlite3/storage.ide differ
diff --git a/src/BTCAnalysis.Forms.MainForm/obj/Debug/BTCAnalysis.Forms.csprojAssemblyReference.cache b/src/BTCAnalysis.Forms.MainForm/obj/Debug/BTCAnalysis.Forms.csprojAssemblyReference.cache
index 3777b78..71e20df 100644
Binary files a/src/BTCAnalysis.Forms.MainForm/obj/Debug/BTCAnalysis.Forms.csprojAssemblyReference.cache and b/src/BTCAnalysis.Forms.MainForm/obj/Debug/BTCAnalysis.Forms.csprojAssemblyReference.cache differ
diff --git a/src/BTCAnalysis/obj/Debug/BTCAnalysis.csprojAssemblyReference.cache b/src/BTCAnalysis/obj/Debug/BTCAnalysis.csprojAssemblyReference.cache
index 9f8e5b9..e5bf139 100644
Binary files a/src/BTCAnalysis/obj/Debug/BTCAnalysis.csprojAssemblyReference.cache and b/src/BTCAnalysis/obj/Debug/BTCAnalysis.csprojAssemblyReference.cache differ
diff --git a/src/ChunkGenerator/obj/Debug/ChunkGenerator.csprojAssemblyReference.cache b/src/ChunkGenerator/obj/Debug/ChunkGenerator.csprojAssemblyReference.cache
index 4026467..3d13c12 100644
Binary files a/src/ChunkGenerator/obj/Debug/ChunkGenerator.csprojAssemblyReference.cache and b/src/ChunkGenerator/obj/Debug/ChunkGenerator.csprojAssemblyReference.cache differ
diff --git a/src/FinanceManager.Console/obj/Debug/FinanceManager.Console.csprojAssemblyReference.cache b/src/FinanceManager.Console/obj/Debug/FinanceManager.Console.csprojAssemblyReference.cache
index dc11a12..8eec964 100644
Binary files a/src/FinanceManager.Console/obj/Debug/FinanceManager.Console.csprojAssemblyReference.cache and b/src/FinanceManager.Console/obj/Debug/FinanceManager.Console.csprojAssemblyReference.cache differ
diff --git a/src/FinanceManager.Forms/FinanceManager.Forms.csproj b/src/FinanceManager.Forms/FinanceManager.Forms.csproj
index 4a2fcf8..2c52bdc 100644
--- a/src/FinanceManager.Forms/FinanceManager.Forms.csproj
+++ b/src/FinanceManager.Forms/FinanceManager.Forms.csproj
@@ -112,6 +112,9 @@
     </Compile>
     <Compile Include="MainForm.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <EmbeddedResource Include="Form1.resx">
+      <DependentUpon>Form1.cs</DependentUpon>
+    </EmbeddedResource>
     <EmbeddedResource Include="Properties\Resources.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
diff --git a/src/FinanceManager.Forms/Form1.Designer.cs b/src/FinanceManager.Forms/Form1.Designer.cs
index ef7726c..d267424 100644
--- a/src/FinanceManager.Forms/Form1.Designer.cs
+++ b/src/FinanceManager.Forms/Form1.Designer.cs
@@ -23,13 +23,66 @@
         /// Der Inhalt der Methode darf nicht mit dem Code-Editor geändert werden.
         /// </summary>
         private void InitializeComponent() {
-            this.components = new System.ComponentModel.Container();
+            this.menuStrip1 = new System.Windows.Forms.MenuStrip();
+            this.dataToolStripMenuItem = new System.Windows.Forms.ToolStripMenuItem();
+            this.extrasToolStripMenuItem = new System.Windows.Forms.ToolStripMenuItem();
+            this.helpToolStripMenuItem = new System.Windows.Forms.ToolStripMenuItem();
+            this.menuStrip1.SuspendLayout();
+            this.SuspendLayout();
+            // 
+            // menuStrip1
+            // 
+            this.menuStrip1.ImageScalingSize = new System.Drawing.Size(20, 20);
+            this.menuStrip1.Items.AddRange(new System.Windows.Forms.ToolStripItem[] {
+            this.dataToolStripMenuItem,
+            this.extrasToolStripMenuItem,
+            this.helpToolStripMenuItem});
+            this.menuStrip1.Location = new System.Drawing.Point(0, 0);
+            this.menuStrip1.Name = "menuStrip1";
+            this.menuStrip1.Size = new System.Drawing.Size(1489, 28);
+            this.menuStrip1.TabIndex = 0;
+            this.menuStrip1.Text = "menuStrip1";
+            // 
+            // dataToolStripMenuItem
+            // 
+            this.dataToolStripMenuItem.Name = "dataToolStripMenuItem";
+            this.dataToolStripMenuItem.Size = new System.Drawing.Size(55, 24);
+            this.dataToolStripMenuItem.Text = "Data";
+            // 
+            // extrasToolStripMenuItem
+            // 
+            this.extrasToolStripMenuItem.Name = "extrasToolStripMenuItem";
+            this.extrasToolStripMenuItem.Size = new System.Drawing.Size(62, 24);
+            this.extrasToolStripMenuItem.Text = "Extras";
+            // 
+            // helpToolStripMenuItem
+            // 
+            this.helpToolStripMenuItem.Name = "helpToolStripMenuItem";
+            this.helpToolStripMenuItem.Size = new System.Drawing.Size(55, 24);
+            this.helpToolStripMenuItem.Text = "Help";
+            // 
+            // Form1
+            // 
+            this.AutoScaleDimensions = new System.Drawing.SizeF(8F, 16F);
             this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
-            this.ClientSize = new System.Drawing.Size(800, 450);
+            this.ClientSize = new System.Drawing.Size(1489, 722);
+            this.Controls.Add(this.menuStrip1);
+            this.MainMenuStrip = this.menuStrip1;
+            this.Name = "Form1";
             this.Text = "Form1";
+            this.menuStrip1.ResumeLayout(false);
+            this.menuStrip1.PerformLayout();
+            this.ResumeLayout(false);
+            this.PerformLayout();
+
         }
 
         #endregion
+
+        private System.Windows.Forms.MenuStrip menuStrip1;
+        private System.Windows.Forms.ToolStripMenuItem dataToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem extrasToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem helpToolStripMenuItem;
     }
 }
 
diff --git a/src/FinanceManager.Forms/obj/Debug/DesignTimeResolveAssemblyReferences.cache b/src/FinanceManager.Forms/obj/Debug/DesignTimeResolveAssemblyReferences.cache
index 9d33d44..1b04c27 100644
Binary files a/src/FinanceManager.Forms/obj/Debug/DesignTimeResolveAssemblyReferences.cache and b/src/FinanceManager.Forms/obj/Debug/DesignTimeResolveAssemblyReferences.cache differ
diff --git a/src/FinanceManager.Forms/obj/Debug/DesignTimeResolveAssemblyReferencesInput.cache b/src/FinanceManager.Forms/obj/Debug/DesignTimeResolveAssemblyReferencesInput.cache
index 6da628c..d5f97d0 100644
Binary files a/src/FinanceManager.Forms/obj/Debug/DesignTimeResolveAssemblyReferencesInput.cache and b/src/FinanceManager.Forms/obj/Debug/DesignTimeResolveAssemblyReferencesInput.cache differ
diff --git a/src/FinanceManager.Forms/obj/Debug/FinanceManager.Forms.csprojAssemblyReference.cache b/src/FinanceManager.Forms/obj/Debug/FinanceManager.Forms.csprojAssemblyReference.cache
index 3777b78..d5381bb 100644
Binary files a/src/FinanceManager.Forms/obj/Debug/FinanceManager.Forms.csprojAssemblyReference.cache and b/src/FinanceManager.Forms/obj/Debug/FinanceManager.Forms.csprojAssemblyReference.cache differ
